### PR TITLE
EMA-2268_Parallel_Run

### PR DESCRIPTION
**Pull request by cgjohn1**

### CODE REVIEW

File: create_cucumber_json.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+sed -i.bak '1s/^/\[/' "target/cucumber/cucumber.json"
+sed -i.bak '$s/$/\]/' "target/cucumber/cucumber.json"
+
+for filename in target/cucumber/*.json; do
+	if [[ $(find $filename -type f -size +4k) ]] && [[ $filename != "target/cucumber/cucumber.json" ]]; then
+       sed -i.bak '$ s/.$//' $filename
+       sed -i".bak" "1s/^/\[/" $filename
+	   sed -i.bak '$s/$/\]/' $filename
+    fi
+done
```
Comment (bravivarman) [inline create_cucumber_json.sh]: Just to understand, This combining cucumber.json is required for reporting isnt?

File: discovery-mobile-app-test.iml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    <orderEntry type="library" name="Maven: com.github.tulskiy:jkeymaster:1.2" level="project" />
+    <orderEntry type="library" name="Maven: net.java.dev.jna:jna:4.2.1" level="project" />
+    <orderEntry type="library" name="Maven: org.slf4j:slf4j-nop:1.7.25" level="project" />
+    <orderEntry type="library" name="Maven: com.melloware:jintellitype:1.3.7" level="project" />
+    <orderEntry type="library" name="Maven: com.nativelibs4java:bridj:0.7.0" level="project" />
+    <orderEntry type="library" name="Maven: com.google.android.tools:dx:1.7" level="project" />
+    <orderEntry type="library" name="Maven: com.sikulix:sikulixlibsmac:1.1.1" level="project" />
+    <orderEntry type="library" name="Maven: com.sikulix:sikulix2tigervnc:2.0.0-SNAPSHOT" level="project" />
+  </component>
+</module>
```
Comment (bravivarman) [inline discovery-mobile-app-test.iml]: Can you remove this?

File: pom.xml [hunk 3]
```diff
@@ -41,6 +47,41 @@
...
+                        <phase>generate-test-sources</phase>
+                        <goals>
+                            <goal>generateRunners</goal>
+                        </goals>
+                        <configuration>
+                            <glue>com.discovery.es</glue>
+                            <featuresDirectory>src/test/resources/features/</featuresDirectory>
+                            <cucumberOutputDir>target/cucumber</cucumberOutputDir>
+                            <format>pretty,json,html,rerun</format>
+                        </configuration>
```
Comment (bravivarman) [inline pom.xml]: Should we add the rerun later?

File: create_cucumber_json.sh (hunk 1 above)
Comment (cgjohn1) [reply create_cucumber_json.sh]: Thats correct

File: discovery-mobile-app-test.iml (hunk 2 above)
Comment (cgjohn1) [reply discovery-mobile-app-test.iml]: It was uploaded for the legacy framework so just thought to do the same. So we dont need this?

File: pom.xml (hunk 3 above)
Comment (cgjohn1) [reply pom.xml]: Well I can remove it but it is disabled in the run_tests script

File: create_cucumber_json.sh (hunk 1 above)
Comment (bravivarman) [reply create_cucumber_json.sh]: Have you tried to use any of the Plugin like - cluecumber-report-plugin that does this merging under the hood?

File: discovery-mobile-app-test.iml (hunk 2 above)
Comment (bravivarman) [reply discovery-mobile-app-test.iml]: Yes. Don't need this. Its IDEA Project file. Can you please update gitIgnore as well.

File: pom.xml (hunk 3 above)
Comment (bravivarman) [reply pom.xml]: Ok

File: create_cucumber_json.sh (hunk 1 above)
Comment (cgjohn1) [reply create_cucumber_json.sh]: No there isn't. All suggestion point to creating something yourself to one file

File: discovery-mobile-app-test.iml (hunk 2 above)
Comment (cgjohn1) [reply discovery-mobile-app-test.iml]: Ok cool

**Pull request merged**